imdclient/IMDClient.py

```diff
@@ -112,13 +120,15 @@ def get_imdframe(self):
             except EOFError:
                 # in this case, consumer is already finished
                 # and doesn't need to be notified
+                # self._wait(self._wait_after_disconnect)
                 self._disconnect()
                 self._stopped = True
                 raise EOFError
         else:
             try:
                 return self._producer._get_imdframe()
             except EOFError:
+                # self._wait(self._wait_after_disconnect)
```

Comment:
Same thing here, no harm in sending the packet as long as try-except mechanism is in place